### PR TITLE
Changing Planning Pipeline Reference

### PR DESCRIPTION
**Pull request by ct2034**
to match https://github.com/ros-planning/moveit/pull/1893

Check releases:
- [ ] https://github.com/ros/rosdistro/pull/30162
- [ ] https://github.com/ros/rosdistro/pull/30168

### CODE REVIEW

**Comment by ct2034** [comment]
TODO: Add info to changelog after https://github.com/PilzDE/pilz_application_templates/pull/13

File: prbt_default_application/CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@ This file lists the changes that you have to trace, once you created an own appl
 and update its dependencies. Date and upstream versions are given for reference, you can check for your local
 version using ``dpkg -l | grep "prbt-\|pilz-"``.
 
+Forthcoming
+-----------
+* compatible with pilz_robots version melodic 0.5.22 and noetic-0.6.0
```
Comment (rfeistenauer) [inline prbt_default_application/CHANGELOG.rst]:
```suggestion
* compatible with pilz_robots version melodic-0.5.22 and noetic-0.6.0
```

File: prbt_default_application/CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@ This file lists the changes that you have to trace, once you created an own appl
 and update its dependencies. Date and upstream versions are given for reference, you can check for your local
 version using ``dpkg -l | grep "prbt-\|pilz-"``.
 
+Forthcoming
+-----------
+* compatible with pilz_robots version melodic-0.5.22 and noetic-0.6.0
```
Comment (martiniil) [inline prbt_default_application/CHANGELOG.rst]:
```suggestion
* compatible with pilz_robots version 0.5.22 (melodic) or 0.6.0 (noetic) and pilz_industrial_motion version 0.4.13 (melodic) or 0.5.0 (noetic)
```

**Review by martiniil — CHANGES_REQUESTED** [review]
See inline comment.

I executed the launch-file with the mentioned versions. It worked.

**Pull request merged**